pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -75,24 +82,31 @@ mkDerivation rec {
     libusb1
     limesuite
     mbelib
+    ninja
     opencv3
     qtcharts
+    qtdeclarative
+    qtgraphicaleffects
     qtlocation
     qtmultimedia
+    qtquickcontrols
+    qtquickcontrols2
```

Comment:
You are sure both?

Reply:
> It builds without it, so I suspect that the doc is incorrect

It does build without any of the dependencies that I have added, you can probably remove most of the ones that are already specified, but then most decoders/plugins don't work. For example now the ADS-B map doesn't work but it builds fine without the required qt dependency.